release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4372,3 +4372,34 @@ $ oc adm release info 4.13.50 --pullspecs
 [id="ocp-4-13-50-updating"]
 ==== Updating
 To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.13.51
+[id="ocp-4-13-51"]
+=== RHSA-2024:6811 - {product-title} 4.13.51 bug fix and security updates
+
+Issued: 25 September 2024
+
+{product-title} release 4.13.51, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:6811[RHSA-2024:6811] advisory. TThe RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2024:6813[RHBA-2024:6813] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.13.51 --pullspecs
+----
+
+[id="ocp-4-13-51-enhancements"]
+==== Enhancements
+
+* CENTOS 8 has recently ended its lifecycle. This release updates the CENTOS 8 references to CENTOS 9. (link:https://issues.redhat.com/browse/OCPBUGS-39160[*OCPBUGS-39160*])
```

Comment:
The [OCPBUGS-39160](https://issues.redhat.com/browse/OCPBUGS-39160) cited here is a 4.14.z bug already shipped/mentioned in the [4.14 RNs](https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html#ocp-4-16-36-enhancements-_release-notes). It looks like [OCPBUGS-41786](https://issues.redhat.com/browse/OCPBUGS-41786) is the 4.13 version of this bug, and appears to be attached to the actual 4.13.z advisory.

Reply:
:heavy_check_mark: